lib/renderers/I18NRenderer.js

```diff
@@ -30,10 +30,14 @@ class I18NRenderer extends DefaultRenderer {
           returnObjects: true,
         });
 
+        if (_.isString(message) && message === result.reply) {
+          throw new Error(`Locale views is not implemented: ${alexaEvent.request.locale}`);
```

Comment:
do we really want to crash in here or do we want the skill to simply fallback to the default language strings? Also, seems like the message is not clear, should be something like `Views for en-gb locale are missing`